handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -1472,6 +1472,9 @@ private int unwrap(ChannelHandlerContext ctx, ByteBuf packet, int length) throws
                     } else {
                         ctx.fireChannelRead(decodeOut);
                     }
+                    if (decodeOut.refCnt() > 0) {
+                        decodeOut.release();
```

Comment:
This is not correct as the ownership is transferred when passing the message to the next handler in the pipeline